cryptostore/data/gd.py

```diff
@@ -121,8 +125,8 @@ def _callback(request_id, response, exception, keep=existing_as_dict):
     def write(self, bucket: str, path: str, file_name: str, **kwargs):
         """
         Upload file to Google Drive. File is stored in a parent folder which
-        name is that of 'path', replacing '/' with '-'. Folder name is
-        generated in `__init__`.
+        name is that of 'path', replacing '/' with `self.folder_name_sep`.
```

Comment:
I'm confused here, I thought the old behavior was separating things with `-` but here you mention you're replacing `/`

Reply:
In the path generated in `parquet.py`, the complete path file is:
`exchange/data_type/pair/exchange-data_type-pair.parquet`

Google Drive API does not understand `exchange/data_type/pair`.
Working with nested folders in Google Drive API implies checking each level separately, so it would be several queries and list processing for not much added value in my opinion.
So instead, in Google Drive, the file path is:
`exchange-data_type-pair/exchange-data_type-pair.parquet`
(so to say, as the notion of path does not exist in Google Drive API)

So, I was saying that the old behavior used `-`, and that now, I want to use `.`. To the user, the file path becomes then.
`exchange.data_type.pair/exchange-data_type-pair.parquet`

In the python docstring, I indicate that to generate the folder name, I am taking the one generated in `parquet.py`:
`exchange/data_type/pair`  (I leave the file name untouch).
And taking this path, I remove the `/` and replace them now with `folder_name_sep`.

I hope this clarifies things.
I did not change my ideas since the beginning, so re-reading what I wrote since the beginning should now make more sense.

Thanks